quickwit/Cargo.toml

```diff
@@ -46,6 +46,7 @@ async-speed-limit = "0.4"
 async-trait = "0.1"
 atty = "0.2"
 backoff = { version = "0.4", features = ["tokio"] }
+base16 = "0.2.1"
```

Comment:
Seems like the `hex` crate is more commonly used and was updated more recently.

Reply:
switched to hex